apps/coscrad-frontend/src/components/nav-menu/nav-menu-item.tsx

```diff
@@ -0,0 +1,19 @@
+import { MenuItem } from '@mui/material';
+import { Link } from 'react-router-dom';
+import { NavItemInfo } from './nav-menu';
+
+export interface NavItemItemProps {
+    index: number;
+    navItemInfo: NavItemInfo;
+    handleClose: () => void;
+}
+
+export const NavMenuItem = ({ index, navItemInfo, handleClose }: NavItemItemProps) => {
```

Comment:
Making index a prop is a huge smell. Don't do this.

Reply:
Use the label as a unique key instead.

Reply:
This is fixed